src/bindings/event.ts

```diff
@@ -17,14 +17,8 @@ export const getBotConfig = () => botConfig;
 let adapters: Adapters = {} as Adapters;
 export const getAdapters = () => adapters;
 
-export type Logger = {
-  info: (msg: string | object, options?: JSON) => void;
-  debug: (msg: string | object, options?: JSON) => void;
-  warn: (msg: string | object, options?: JSON) => void;
-  error: (msg: string | object, options?: JSON) => void;
-};
-let logger: Logger;
-export const getLogger = (): Logger => logger;
+let logger: GitHubLogger;
+export const getLogger = (): GitHubLogger => logger;
```

Comment:
Revert this and inherit GithubLogger from Logger.

Reply:
> Revert this and inherit GithubLogger from Logger.

I'd have to make `Logger` an interface so I can use `implements`.. you cannot extend types